CONTRIBUTING.md

```diff
@@ -0,0 +1,64 @@
+# Contributing to the Federal Front Door Microsite
+
+We aspire to create a welcoming environment for collaboration on this project.
+To that end, we follow the [18F Code of Conduct] [18f-coc] and ask that all
+contributors do the same.
+
+[18f-coc]: https://github.com/18F/code-of-conduct/blob/master/code-of-conduct.md "18F: Code of Conduct"
+
+ Section | Description
+ ------- | -----------
+ [Opening an Issue](#opening-an-issue) | Information on contributing Issues
+ [Submitting Pull Requests](#submitting-pull-requests) | Information on contributing Pull Requests
+ [Deployment Cycle](#deployment-cycle) | Information on the deployment cycle
+
+## Opening an Issue
+
+Before submitting a new Issue, please consider the following guidelines:
+
+- Search [this repository for open or closed Issues] [gh-search-issue] that may
+  relate to your Pull Request.
+
+To help us get a better understanding of the issue you are submitting, please
+leverage the following outline (as used in the following
```

Comment:
Unless you all strongly object, I'd recommend replacing "leverage" with "use" — the latter is more plain-language friendly.